packages/sampler/stories/qa/VirtualGridList.js

```diff
@@ -58,17 +64,20 @@ storiesOf('VirtualGridList', module)
 			<VirtualGridList
 				dataSize={updateDataSize(number('dataSize', Config, defaultDataSize))}
 				direction="horizontal"
-				focusableScrollbar={boolean('focusableScrollbar', Config, false)}
-				horizontalScrollbar={select('horizontalScrollbar', ['auto', 'hidden', 'visible'], Config, 'auto')}
+				focusableScrollbar={boolean('focusableScrollbar', Config)}
+				horizontalScrollbar={select('horizontalScrollbar', ['auto', 'hidden', 'visible'], Config)}
 				itemRenderer={renderItem}
 				itemSize={{
 					minWidth: ri.scale(number('minWidth', Config, 180)),
 					minHeight: ri.scale(number('minHeight', Config, 270))
 				}}
+				noScrollByWheel={boolean('noScrollByWheel', Config)}
 				onKeyDown={action('onKeyDown')}
 				onScrollStart={action('onScrollStart')}
 				onScrollStop={action('onScrollStop')}
 				spacing={ri.scale(number('spacing', Config, 18))}
+				spotlightDisabled={boolean('spotlightDisabled', Config, false)}
```

Comment:
`verticalScrollbar` prop is missed.

Reply:
The QA VirtualGridList sample is supported to the only horizon case. So, I thought `verticalScrollbar` is not needed.